app/components/organisms/ComplimentaryRegion/index.js

```diff
@@ -1,7 +1,8 @@
 import styled from 'styled-components';
 
 const ComplimentaryRegion = styled.div`
-  margin-top: 1rem;
+  padding-left: 16px;
```

Comment:
This is probably 1rem? :D 
```suggestion
  padding-left: 1rem;
```